dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -112,9 +137,42 @@ COPY --from=home {{ $beatHome }}/NOTICE.txt /licenses
 {{- if ne .user "root" }}
 RUN groupadd --gid 1000 {{ .BeatName }}
 RUN useradd -M --uid 1000 --gid 1000 --groups 0 --home {{ $beatHome }} {{ .user }}
+{{- if (and (eq .Variant "offline") (not (contains .from "ubi-minimal")))  }}
+RUN chown {{ .user }} $NODE_PATH
+{{- end }}
 {{- end }}
 USER {{ .user }}
 
+{{- if (and (eq .Variant "offline") (not (contains .from "ubi-minimal")))  }}
+# Setup synthetics env vars
+ENV ELASTIC_SYNTHETICS_CAPABLE=true
+ENV SUITES_DIR={{ $beatHome }}/suites
```

Comment:
Are we using a dedicated folder instead of `/tmp` for copying suites? I forgot what wat was the real use of this flag.

Reply:
This can probably be nixed. It's not really used for anything.